Assets/Scripts/FeedingSystem/DogFoodBowl.cs

```diff
@@ -102,6 +105,27 @@ public void FeedDogs()
         }
     }
 
+    public void FeedDogsS()
```

Comment:
This comment applies across a majority of the changes here, but we should be copying the functionality for each type of food. Instead, you should expand the Feed method and others like it to take a food or a type of food as an argument. This should cut down on redundant code